gcsio/src/main/java/com/google/cloud/hadoop/gcsio/FileInfo.java

```diff
@@ -69,6 +69,13 @@ public boolean isDirectory() {
     return itemInfo.isDirectory();
   }
 
+  /**
```

Comment:
Use https://github.com/google/google-java-format to format changed lines

Reply:
Done